projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb

```diff
@@ -0,0 +1,2293 @@
+{
```

Comment:
<p>I am not sure the dfs.head() output here is different from the one 4 cells above. If the point is to verify that the variables are categories, another way might be better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Yes, you are right! I'll change this.</p>

Reply:
<p>Here what I'm using to check the types is the <code>dtypes</code> attribute, so I'll just use that again, and remove the extra call to <code>.head()</code> </p>